lambdaclass/mirra_backend
# Add two additional spikes

> [!Warning]
> Merge alongside [This client pr](https://github.com/lambdaclass/champions_of_mirra/pull/1927)

## Motivation

We want two more spikes

## Summary of changes

- Add two spikes in the config file

## How to test it?

Start a match going to [This link](http://localhost:3000/)  and you should 4 obstacles changing colors, you can try walking over the two closest to the middle when they are red and you should be able to walk over them

## Checklist
